modern/src/App.js

```diff
@@ -22,6 +22,11 @@ async function getSkin() {
   return await initialize(zip, skinXml);
 }
 
+function setupUpdates(node) {
```

Comment:
Can we rename to `useJsUpdates`? Having the “use” prefix is a strong convention for hooks.

Reply:
Makes sense! 👍